rita_common/src/peer_listener/mod.rs

```diff
@@ -83,14 +82,57 @@ impl Default for PeerListener {
 impl PeerListener {
     pub fn new() -> Result<PeerListener, RitaCommonError> {
         Ok(PeerListener {
-            contacting_neighbors: false,
             interfaces: HashMap::new(),
             peers: HashMap::new(),
             interface_map: HashMap::new(),
         })
     }
 }
 
+impl Clone for PeerListener {
```

Comment:
Do we have to clone all of peer listener or could we get away with grabbing a copy of just the interfaces for sending new hellos?